parachain/pallets/commitments/src/lib.rs

```diff
@@ -98,6 +105,8 @@ impl<T: Trait> Commitments for Module<T> {
 
 	// Add a message for eventual inclusion in a commitment
 	fn add(address: H160, payload: Vec<u8>) {
-		<Self as Store>::Messages::append(Message { address, payload });
+		let nonce = <Self as Store>::Nonce::get();
+		<Self as Store>::Messages::append(Message { address, payload, nonce });
+		<Self as Store>::Nonce::set(nonce + 1);
```

Comment:
Is storing 

a) the next nonce to be used or 
b) the last nonce that has been used

the canonical way? CC @denalimarsh @hcaw